awx/ui/client/src/management-jobs/card/card.controller.js

```diff
@@ -48,66 +48,84 @@ export default
                 $scope.submitJob = function (id, name) {
                     Wait('start');
                         defaultUrl = GetBasePath('system_job_templates')+id+'/launch/';
-                        CreateDialog({
-                            id: 'prompt-for-days'    ,
-                            title: name,
-                            scope: $scope,
-                            width: 500,
-                            height: 300,
-                            minWidth: 200,
-                            callback: 'PromptForDays',
-                            resizable: false,
-                            onOpen: function(){
-                                $scope.$watch('prompt_for_days_form.$invalid', function(invalid) {
-                                    if (invalid === true) {
-                                        $('#prompt-for-days-launch').prop("disabled", true);
-                                    } else {
-                                        $('#prompt-for-days-launch').prop("disabled", false);
-                                    }
-                                });
+                        var noModalJobs = ['Automation Insights Collection', 'Cleanup Expired Sessions', 'Cleanup Expired OAuth 2 Tokens']
```

Comment:
we need to verify that these strings are never translated on the backend, otherwise we can't use them for this purpose.

Reply:
I just confirmed that these names will not be translated by the django translations backend.